browser-test/src/applicant/navigation/northstar_application_navigation_with_eligibility_conditions.test.ts

```diff
@@ -205,117 +222,54 @@ test.describe('Applicant navigation flow', {tag: ['@northstar']}, () => {
           fullProgramName,
-          /* isEligible= */ false,
-        )
-      })
-
-      await test.step('answer the other question', async () => {
-        await applicantQuestions.clickApplyProgramButton(fullProgramName)
-        await applicantQuestions.answerEmailQuestion('email@email.com')
-      })
-
-      await test.step("submit and expect to be told it's ineligible", async () => {
-        await applicantQuestions.clickContinue()
-        await applicantQuestions.expectMayNotBeEligibileAlertToBeVisible()
-        await applicantQuestions.expectIneligibleQuestionInReviewPageAlert(
-          AdminQuestions.NUMBER_QUESTION_TEXT,
-        )
-        await applicantQuestions.clickSubmitApplication()
-        await applicantQuestions.expectMayNotBeEligibileAlertToBeVisible()
-        await applicantQuestions.expectIneligibleQuestionInReviewPageAlert(
-          AdminQuestions.NUMBER_QUESTION_TEXT,
-        )
-      })
-    })
-
-    test('ineligible page renders markdown', async ({
-      page,
-      adminQuestions,
-      applicantQuestions,
-      adminPredicates,
-      adminPrograms,
-    }) => {
-      const questionName = 'question-with-markdown'
-      const programName = 'Program with markdown question'
-
-      await test.step('Set up program with markdown in question text', async () => {
-        await loginAsAdmin(page)
-        await adminQuestions.addTextQuestion({
-          questionName: questionName,
-          questionText:
-            'This is a _question_ with some [markdown](https://www.example.com) and \n line \n\n breaks',
-          markdown: true,
-        })
-        await adminPrograms.addProgram(programName)
-        await adminPrograms.editProgramBlock(programName, 'first description', [
-          questionName,
-        ])
-        // Add an eligiblity condition on the markdown question
-        await adminPrograms.goToEditBlockEligibilityPredicatePage(
-          programName,
-          'Screen 1',
-        )
-        await adminPredicates.addPredicates({
-          questionName: questionName,
-          scalar: 'text',
-          operator: 'is equal to',
-          value: 'foo',
-        })
-        await adminPrograms.gotoAdminProgramsPage()
-        await adminPrograms.publishProgram(programName)
-        await logout(page)
-      })
-
-      await test.step('apply to program with ineligible answer and verify text', async () => {
-        await applicantQuestions.applyProgram(
-          programName,
-          /* northStarEnabled= */ true,
-        )
-        await applicantQuestions.answerTextQuestion('bar')
-        await applicantQuestions.clickContinue()
-        await applicantQuestions.expectIneligiblePage(
-          /* northStarEnabled= */ true,
+          /* isEligible= */ true,
         )
         await validateScreenshot(
           page,
-          'northstar-ineligible-page-with-markdown',
+          'eligible-home-page-program-tag-gating-eligibility',
```

Comment:
it's the only one in this test of the eligible tag, and since we remove eligibility tags from in-progress applications now it might be the only one in genearl, but I haven't confirmed. I'll narrow it and the other screenshots in this test, though.

Reply:
Since we have the mobile, tablet and desktop screenshots for the program card with the ineligible tag, I feel like we don't need all three screenshots for the card with the eligible tag.  Just trying to avoid so many screenshots since it slows down the tests.